modules/@apostrophecms/page/index.js

```diff
@@ -2187,7 +2188,47 @@ database.`);
           }
         }
       },
-      ...require('./lib/legacy-migrations')(self)
+      ...require('./lib/legacy-migrations')(self),
+      addMisreplicatedParkedPagesMigration() {
+        self.apos.migration.add('misreplicated-parked-pages', async () => {
+          const parkedPages = await self.apos.doc.db.find({
+            parkedId: {
+              $exists: 1
+            }
+          }).toArray();
+          let locales = Object.keys(self.apos.i18n.locales);
+          if (locales[0] !== self.apos.i18n.defaultLocale) {
+            locales = [ self.apos.i18n.defaultLocale, ...locales.filter(locale => locale !== self.apos.i18n.defaultLocale) ];
+          }
+          const parkedIds = [ ...new Set(parkedPages.map(page => page.parkedId)) ];
```

Comment:
Aren't `parkedIds` supposed to be unique?

Reply:
They are unique within a locale + mode combo, but intentionally not so across locales, as there's always a draft for every published version, and (for parked pages, not everything) always an fr for every en, etc.